site/components/DateSelector.tsx

```diff
@@ -1,81 +1,124 @@
-                    >
-                        Month
-                    </label>
-                    <input
-                        className={`govuk-input govuk-date-input__input govuk-input--width-2 ${
-                            errors.length > 0 ? "govuk-input--error" : ""
-                        } `}
-                        id={`${type}-${startOrEnd}-month-input`}
-                        name={`${type}${type ? startCase(startOrEnd) : startOrEnd}DateMonth`}
-                        type="text"
-                        defaultValue={inputs?.monthInput}
-                    />
-                </div>
-            </div>
-            <div className="govuk-date-input__item">
-                <div className="govuk-form-group">
-                    <label
-                        className="govuk-label govuk-date-input__label"
-                        htmlFor={type ? `${type}-${startOrEnd}-year-input` : `${startOrEnd}-year-input`}
-                    >
-                        Year
-                    </label>
-                    <input
-                        className={`govuk-input govuk-date-input__input govuk-input--width-4 ${
-                            errors.length > 0 ? "govuk-input--error" : ""
-                        } `}
-                        id={`${type}-${startOrEnd}-year-input`}
-                        name={`${type}${type ? startCase(startOrEnd) : startOrEnd}DateYear`}
-                        type="text"
-                        defaultValue={inputs?.yearInput}
-                    />
-                </div>
+const inputBox = (
+    startOrEnd: "start" | "end",
+    hasErrors: boolean,
+    inputRef: React.Ref<any> | undefined,
+    inputProps: InputBaseComponentProps | undefined,
+    InputProps: Partial<FilledInputProps> | Partial<OutlinedInputProps> | undefined,
+    disabled: boolean,
+    isPublishInfo: boolean
+) => (
+    <div className="govuk-date-input flex items-center" id={`${startOrEnd}-date`}>
+        <div className="govuk-date-input__item govuk-!-margin-right-0">
+            <div className="govuk-form-group">
+                <input
+                    className={`govuk-input govuk-date-input__input govuk-input--width-6 ${
+                        hasErrors ? "govuk-input--error" : ""
+                    } `}
+                    id={`${isPublishInfo && 'publish-'}${startOrEnd}-day-input`}
+                    name={`${startOrEnd}DateDay`}
+                    type="text"
+                    ref={inputRef}
+                    {...inputProps}
+                    disabled={disabled}
+                />
             </div>
         </div>
-    </FormElementWrapper>
+        {InputProps?.endAdornment}
+    </div>
 );
 
+const renderWeekPickerDay = (
+    _date: Date,
+    _selectedDates: Array<Date | null>,
+    pickersDayProps: PickersDayProps<Date>,
+) => {
+    return (
+        <PickersDay
+            {...pickersDayProps}
+            sx={{
```

Comment:
Looks like material ui lets you add classes to the components as well so we could use tailwind directly here with something like:
```
classes={{
    selected: "!bg-govBlue",
    dayWithMargin:
        "focus:!border focus:!border-solid hover:!border hover:!border-solid hover:!border-govBlue focus:!border-govYellow",
    }}
```

Reply:
added